packages/picasso.js/src/core/chart-components/text/text.js

```diff
@@ -116,6 +119,17 @@ function generateTitle({ title, definitionSettings, dock, rect, measureText, sty
  * @property {string} [join=', '] - String to add when joining titles from multiple sources
  * @property {number} [maxLengthPx] - Limit the text length
  */
+
+/**
+ * @typedef {object} ComponentText.style.text
+ * @property {string} [fontSize='12px'] - Font size of text
+ * @property {string} [fontFamily='Source Sans Pro'] - Font family of text
+ * @property {string} [fontWeight='bold'] - Font weight of the overlay text
+ * @property {string} [fill='#ffffff'] - Fill color of the overlay text
+ * @property {string} [stroke='#595959'] - Stroke of the overlay text
+ * @property {number} [strokeWidth=2] - Stroke width of the overlay text
+ * @property {number} [opacity=0.5] - Opacity of the overlay text
```

Comment:
Should remove the word `overlay` from all of these.

Reply:
Yes